test/Driver/parseable_output.swift

```diff
@@ -1,44 +1,42 @@
 // RUN: %swiftc_driver_plain -emit-executable %s -o %t.out -emit-module -emit-module-path %t.swiftmodule -emit-objc-header-path %t.h -serialize-diagnostics -emit-dependencies -parseable-output -driver-skip-execution 2>&1 | %FileCheck %s
 
-// XFAIL: freebsd, linux
```

Comment:
Does this work now?

Reply:
Nope :( That shouldn't have been removed.